app/routes/preflight.py

```
@@ -0,0 +1,552 @@
+
+    return result
+
+
+@router.get(
+    "/status",
+    response_model=PreflightStatusResponse,
+    summary="Current service status",
+    description="Returns current status of all services using cached values"
+)
+async def get_service_status(
+    orchestrator: ServiceOrchestrator = Depends(get_service_orchestrator)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)